.circleci/config.yml

```diff
@@ -1,8 +1,53 @@
 version: 2.1
 
+orbs:
+  go: circleci/go@1.7.1
+
+parameters:
+  kustomize-version:
+    type: string
+    default: v4.5.7
+
 jobs:
-  build:
+  build-and-push:
     docker:
-     - image: cimg/base:2022.05
+     - image: cimg/base:2022.08
+    resource_class: medium
     steps:
-        - run: echo "Sample CircleCI job."
+      - go/install:
```

Comment:
Is this `go` being used for anything? The builds seem to happen in a docker container.

Reply:
the build is actually not happening in a container, it first compiles the go and then builds an image, so this is needed to have go available for compiling/tests/packages